lib/widgets/content.dart

```diff
@@ -946,11 +967,74 @@ class _KatexSpan extends StatelessWidget {
         child: widget);
     }
 
-    return SizedBox(
+    final margin = switch ((styles.marginLeftEm, styles.marginRightEm)) {
+      (null, null) => null,
+      (null, final marginRightEm?) =>
+        EdgeInsets.only(right: marginRightEm * em),
+      (final marginLeftEm?, null) =>
+        EdgeInsets.only(left: marginLeftEm * em),
+      (final marginLeftEm?, final marginRightEm?) =>
+        EdgeInsets.only(left: marginLeftEm * em, right: marginRightEm * em),
+    };
+
+    widget = SizedBox(
       height: styles.heightEm != null
-        ? styles.heightEm! * (fontSize ?? em)
+        ? styles.heightEm! * em
         : null,
       child: widget);
+
+    if (margin != null) {
```

Comment:
nit: put the `margin` computation above next to the spot where `margin` is used, here